### PR TITLE
Expose link color as a custom CSS property

### PR DESCRIPTION
**Pull request by justincy**
Exposes `--fs-color-blue-link` and `--fs-color-purple-link-visited` as CSS custom properties

### CODE REVIEW

**Comment by timothydm42** [comment]
Looks good, but why not put them in the colors section of variables.styl ? Maybe the 'other' section..

**Comment by justincy** [comment]
Good question. Looks like my choice was pretty random. I don't know what I was thinking. You prefer having them in the color section?

**Comment by timothydm42** [comment]
Yes please, I'd say go ahead and put them in the 'other colors' section, with a comment bloc like the other colors have, changed to describe your colors. 

also, keep the naming convention we have; so I'd name the colors 'color-blue-link' and 'color-blue-link-visited' assuming they're blue

**Comment by justincy** [comment]
Your wish is my command.

I made the changes.

File: assets/css/helpers/_variables.styl [hunk 1]
```diff
@@ -627,10 +627,32 @@ $color-pink-gender-light = #f48eb6;
  * @colordef Grey Border
  * @color {hex} #CCCCCC
  * @color {rgb} 204,204,204
- * @colorof Colors.Other Colors.Border Color
+ * @colorof Colors.Other Colors.Border Colors
```
Comment (timothydm42) [inline assets/css/helpers/_variables.styl]: thanks

File: assets/css/helpers/_variables.styl [hunk 2]
```diff
@@ -694,8 +716,7 @@ $line-height-large-xx = 2.15rem;
 $line-height-large-xxx = 2.5rem;
 $line-height-large-xxxx = 2.875rem;
 
-
-
+// link colors
```
Comment (timothydm42) [inline assets/css/helpers/_variables.styl]: erase this :)

File: dist/familysearch-styles.css [hunk 3]
```diff
@@ -1548,7 +1548,7 @@ input.fs-field-group__input ~ .fs-field-group__attachment {
   line-height: 1;
 }
 .fs-person--person-card a.fs-person-vitals__name-full:visited {
-  color: #0060d7;
+  color: $link-blue;
```
Comment (timothydm42) [inline dist/familysearch-styles.css]: This is a generated file, so I think this should correct itself if you run gulp build then push again?

Comment (justincy) [reply dist/familysearch-styles.css]: That was actually an error. I missed one occurrence of `$link-blue`.

File: assets/css/helpers/_variables.styl (hunk 2 above)
Comment (justincy) [reply assets/css/helpers/_variables.styl]: Done.

**Comment by justincy** [comment]
Will there be a new release for this or are you waiting to do a new release until you add something else this iteration?

**Comment by timothydm42** [comment]
@justincy I'll add it into some other release stuff that we're doing this iteration. Thanks for reminding me

**Pull request merged**